### PR TITLE
doc: update acrnd/acrnctl option docs

### PR DESCRIPTION
**Pull request by dbkinder**
Document the acrnd -t, and acrnctl add -C options, noted as missing in
PR #3137

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

File: tools/acrn-manager/README.rst [hunk 1]
```diff
@@ -48,6 +48,11 @@ script that will launch a UOS, for example ``launch_UOS.sh``:
    # acrnctl add launch_UOS.sh -U 1
```
Comment (dbkinder) [inline tools/acrn-manager/README.rst]: @taoyuhong @gvancuts @NanlinXie I don't see a "-U" option supported in the acrnctl.c code?  Maybe it was, but is this command example incorrect now?

Comment (taoyuhong) [reply tools/acrn-manager/README.rst]: The '-U 1' is the optional args of launch_UOS.sh.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: tools/acrn-manager/README.rst (hunk 1 above)
Comment (dbkinder) [reply tools/acrn-manager/README.rst]: Ah. OK.  I was thinking the ``-U 1`` was part of the ``acrnctl add`` command

**Pull request merged**